CHANGES.md

```diff
@@ -1,3 +1,8 @@
+## v0.5.0
+* Add multi-node support for Bitbucket, Confluence and Jira
+* Verification of Jira Helm deployments to Azure AKS
+* Additional bug fixes and stability improvements
+* Additional improvements to logging and testing
```

Comment:
I think we can mention that optionally you can create elk stack with fluentd logger.